StopSelector/index.html

```diff
@@ -38,12 +51,12 @@ <h3 class="text-center or">----Or----</h3>
     </div>
     <div class="row selector-holder">
       <div class="col-md-6">
-        <h2>Filter by routes</h2>
-        <select name="routes[]" class="routes" data-placeholder="Select Routes" multiple></select>
+        <h2>Select Routes</h2> <font color="white">&nbsp;[Optional]</font>
```

Comment:
<font> is an [obsolete tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font). Use CSS instead.  Also, `&nbsp;` should be avoided in general unless you have a very specific reason why

```
Select Routes
[Optional]
```

is unacceptable line-breaking